tests/modeldiffs/fastmri/compare.py

```diff
@@ -50,6 +50,7 @@ def sort_key(k):
   return sd
 
 
+key_transform = None
```

Comment:
I added this because I am importing the transform functions within the traindiff class and i do not transform all the keys always -- for example, it imports from the fastmri/compare.py when the workload being tested is fastmri.